app/src/main/java/org/oppia/android/app/onboarding/OnboardingAppLanguageViewModel.kt

```diff
@@ -0,0 +1,28 @@
+package org.oppia.android.app.onboarding
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import org.oppia.android.app.model.OppiaLanguage
+import org.oppia.android.app.viewmodel.ObservableViewModel
+import javax.inject.Inject
+
+/** ViewModel for managing language selection in [OnboardingFragment]. */
+class OnboardingAppLanguageViewModel @Inject constructor() : ObservableViewModel() {
+  /** The selected app language displayed in the language dropdown. */
+  val languageSelectionLiveData: LiveData<OppiaLanguage> get() = _languageSelectionLiveData
+  private val _languageSelectionLiveData = MutableLiveData<OppiaLanguage>()
+
+  /** Get the list of app supported languages to be displayed in the language dropdown. */
+  val supportedAppLanguagesList: LiveData<List<String>> get() = _supportedAppLanguagesList
```

Comment:
Why isn't this a list of ``OppiaLanguage``s? I saw your other comment on this (the thread is now closed since it was focused on the other property in this view model)--is it possible to keep this as a list of ``OppiaLanguage``s for consistency and perform the conversion at binding time like we do for the language selection?

Reply:
I initially didn't update this viewmodel(the other moldifications were in the AudioLanguageViewModel) since I didn't need a binding adapter here(only one source of truth). But per this comment I have updated to OppiaLanguage for consistency.